worlds/hk/Options.py

```diff
@@ -132,7 +132,8 @@
 for option_name, option_data in pool_options.items():
     extra_data = {"__module__": __name__, "items": option_data[0], "locations": option_data[1]}
     if option_name in option_docstrings:
-        extra_data["__doc__"] = option_docstrings[option_name]
+        extra_data["__doc__"] = option_docstrings[option_name] + \
+            f"\n    This option adds {len(option_data[0])} location(s)"
```

Comment:
You could do this (also adds a missing final period):
```suggestion
            f"\n    This option adds {len(option_data[0])} location{'' if len(option_data[0]) == 1 else 's'}."
```